src/replica/replica.h

```diff
@@ -127,6 +127,7 @@ class replica : public serverlet<replica>, public ref_counter, public replica_ba
     void update_throttle_env_internal(const std::map<std::string, std::string> &envs,
                                       const std::string &key,
                                       throttling_controller &cntl);
+    void update_ac_allowed_users(const std::map<std::string, std::string> &envs);
```

Comment:
I think use 'access' instead of 'ac' is ok, not good to short partial words of a function name.

Reply:
`ac` means `access_controller`, but it's too long if we use `update_access_controller_allowed_users`

Reply:
`ac` is acceptable to me. I'd recommend adding a comment above the function declaration.

A must-do is to validate this env here: https://github.com/XiaoMi/rdsn/blob/master/src/meta/app_env_validator.cpp.

Reply:
I agree with @neverchanje , I will expect adding comment for this function in this pull request and the validation of `allowed_user` in your further pull request.